### PR TITLE
[flutter_tool] Fill in Fuchsia version string

### PR DESCRIPTION
**Pull request by zanderso**
## Description

Pull a version string for Fuchsia from the device.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from our [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart [hunk 1]
```diff
@@ -350,7 +350,20 @@ class FuchsiaDevice extends Device {
   Future<TargetPlatform> get targetPlatform async => TargetPlatform.fuchsia;
 
   @override
-  Future<String> get sdkNameAndVersion async => 'Fuchsia';
+  Future<String> get sdkNameAndVersion async {
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart]: tested: manual :)

**Review by dnfield — APPROVED** [review]
I want to ask for a test, but .. I also realize that by the time we got a test for this it could very well change again.

LGTM.

**Review by jonahwilliams — APPROVED** [review]
LGTM!

**Comment by zanderso** [comment]
Added tests.

**Pull request merged**